pandas/tests/io/test_excel.py

```diff
@@ -1212,6 +1212,17 @@ class and any subclasses, on account of the `autouse=True`
 class TestExcelWriter(_WriterBase):
     # Base class for test cases to run with different Excel writers.
 
+    # def test_excel_sheet_size(self):
+    #     assert False
+        
+    #     BREAKING_SHAPE = (2**20 + 1, 2**14 + 1)
+    #     arr = np.zeros(shape=BREAKING_SHAPE)
+    #     df = pd.DataFrame(arr)
+    #     filepath = 'test.xlsx'
+
+    #     with pytest.raises(ValueError):
```

Comment:
Can you match the expected message? We do this in a lot of other places

Reply:
check.